Tests/TuistKitTests/ImportFinder/ImportFinderTests.swift

```diff
@@ -0,0 +1,75 @@
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistKit
+
+final class ImportFinderTests: TuistUnitTestCase {
```

Comment:
nice - those tests are quite handy 👌

A few additional cases to consider are

* Swift multiple single line imports

```swift
import ModuleA; import ModuleB
```

* Swift import explicit clang modules

```
import ModuleA.Submodule
```

* Swift type imports

```swift
import struct ModuleA.SomeStruct
import enum ModuleA.SomeEnum
import class ModuleA.SomeClass
```

* Clang multiple single line imports

```objc
@import ModuleA; @import ModuleB;
```

* clang import explicit modules

```objc
@import ModuleA.Submodule;
```

Reply:
I added test to cover them all. If i forget something, tell me please